DiffPlex/DiffBuilder/SideBySideDiffBuilder.cs

```diff
@@ -104,6 +104,59 @@ public static SideBySideDiffModel Diff(IDiffer differ, string oldText, string ne
             return model;
         }
 
+        public static SideBySideDiffModel Diff(
```

Comment:
Couldn't we update the other methods in this file to use the new more general method here?

Reply:
Good point If unit tests will still pass I'll look into this.